pkg/logagent/logsagent.go

```diff
@@ -66,4 +67,9 @@ func Start() {
 
 	s := tailer.New(config.GetLogsSources(), filePipelinesEntryChannels, a)
 	s.Start()
+
+	// fixme: this is not great, we share the same pipelines as tailers.
+	// they can easily get overloaded, we'd need a better design
+	c := container.New(config.GetLogsSources(), filePipelinesEntryChannels, a)
+	c.Start()
```

Comment:
let's either address this fixme, or rephrase it (more likely rephrase it and queue some work)